docs/contracts/constant-state-variables.rst

```diff
@@ -6,7 +6,8 @@
 Constant and Immutable State Variables
 **************************************
 
-State variables can be declared as ``constant`` or ``immutable``.
+State variables located in storage can be declared as ``constant`` or ``immutable``,
```

Comment:
Neither immutables nor constants are located in storage :-).
So maybe just
```suggestion
State variables can be declared as one of ``constant`` or ``immutable`` or ``transient``.
```
Or something like that.